Compiler warning in checkercomponent-ti.hpp

#5988 introduces a new compiler warning:

```
-- Build files have been written to: /Users/gunnar/icinga2/build
[26/37] Building CXX object lib/checker/CMakeFiles/checker.dir/checker_unity.cpp.o
In file included from lib/checker/checker_unity.cpp:1:
In file included from ../lib/checker/checkercomponent.cpp:20:
In file included from ../lib/checker/checkercomponent.hpp:23:
lib/checker/checkercomponent-ti.hpp:74:6: warning: private field 'm_ConcurrentChecks' is not used [-Wunused-private-field]
        int m_ConcurrentChecks;
            ^
1 warning generated.
[37/37] Linking CXX executable Bin/Debug/icinga2
```

--- a/lib/checker/checkercomponent.ti
+++ b/lib/checker/checkercomponent.ti
@@ -28,7 +28,7 @@ class CheckerComponent : ConfigObject
 {
 	activation_priority 100;
 
-	[config] int concurrent_checks {
+	[config, no_storage] int concurrent_checks {
 		get {{{
 			return Application::GetMaxConcurrentChecks();
 		}}}

